src/main/drivers/barometer/barometer_bmp085.h

```diff
@@ -25,15 +25,15 @@
 #define BMP085_I2C_ADDR         0x77
 
 typedef struct bmp085Config_s {
-    ioTag_t xclrIO;
-    ioTag_t eocIO;
+    ioTag_t xclrTag;
+    ioTag_t eocTag;
 } bmp085Config_t;
 
 bool bmp085Detect(const bmp085Config_t *config, baroDev_t *baro);
 void bmp085Disable(const bmp085Config_t *config);
 
-#if defined(BARO_EOC_GPIO)
-bool bmp085TestEOCConnected(const bmp085Config_t *config);
+#if defined(BARO_EOC_PIN)
```

Comment:
This will not work as intended - BARO_EOC_PIN is always defined since the resource is dynamic now.